numbers/curveFit/curveFit.go

```diff
@@ -0,0 +1,8 @@
+package curveFit
```

Comment:
this file will probably be deleted but I'm still working on how to do the sampling and fitting that I need to do